next.config.js

```diff
@@ -7,6 +11,70 @@ const nextConfig = {
   }
 }
 
+// API endpoints that should be forwarded to the legacy site
+const legacyAPIPaths = [
+  '/total-ogn',
+  '/total-ogv',
+  '/total-ousd',
+  '/circulating-ogn',
+  '/circulating-ogv',
+  '/social-stats',
+  '/mailing-list/:action*',
+]
+
+// Map legacy routes to new ones here
+const legacyPageMappings = [
+  // [source, destination]
+  ['/dashboard', '/'],
+  ['/presale', '/'],
+  ['/tokens', '/'],
+  
+  ['/product', '/'],
+  ['/ogn-token', '/'],
```

Comment:
This should redirect to /dashboard instead of / (or whatever the future OGN dashboard URL is).

Reply:
Right now, dashboard also redirects to `/`, so I guess we will have to wait until we build out those pages